permissions/plugin-aws-credentials.yml

```diff
@@ -18,6 +18,7 @@ developers:
   - "kerogers"
   - "aneveux"
   - "ajard"
+  - "@cloudbees-developers"
```

Comment:
see line 17?  this is already present

![image](https://github.com/user-attachments/assets/061e340f-e6c9-4cf5-b942-74a11eb893f0)

Reply:
Sorry, fixed in 748604d31cd337d67d5c37d848e3da2928a0856f